### PR TITLE
Dev 4.0.1

### PR DESCRIPTION
**Pull request by behrensger**
This small patch enables the Extension Properties from Camunda 7.15.
The interface description is here https://docs.camunda.org/manual/7.15/reference/rest/external-task/fetch/ (grep for "includeExtensionProperties" on this side)

The extension properties are reachable in the Camunda Modeller Tool as a tab in the Properties Panel. I use these properties to model a kind of constants in the workflow.

Interestingly, your code already works with properties, the only thing missing is the message to Camunda "send the properties" as part of a task. I also added some support stuff to simplify the access to these properties.

includeExtensionProperties=True means that the  properties will be sent by default for people who use the default config and won't be sent for older projects with an overridden configuration.

### CODE REVIEW

File: camunda/variables/properties.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Properties:
+    def __init__(self, properties={}):
+        self.properties = properties
+
+    def get_property(self, property_name):
+        prop = self.properties.get(property_name, None)
+        if not prop:
+            return None
+
+        return prop
```
Comment (yogeshrnaik) [inline camunda/variables/properties.py:10]:
```suggestion
        return self.properties.get(property_name, None)
```
this should be sufficient. not need to check None

**Review by yogeshrnaik — APPROVED** [review]
Approved with one comment

**Review by yogeshrnaik — CHANGES_REQUESTED** [review]
Pls fix one review comment and failed test case

File: camunda/client/external_task_client.py [hunk 2]
```diff
@@ -66,7 +67,11 @@ def _get_topics(self, topic_names, process_variables):
             topics.append({
                 "topicName": topic,
                 "lockDuration": self.config["lockDuration"],
-                "processVariables": process_variables if process_variables else {}
+                "processVariables": process_variables if process_variables else {},
+                # enables Camunda Extension Properties
+                "includeExtensionProperties": self.config["includeExtensionProperties"]
+                if "includeExtensionProperties" in self.config else False
```
Comment (yogeshrnaik) [inline camunda/client/external_task_client.py:73]:
```suggestion
                "includeExtensionProperties": self.config.get("includeExtensionProperties") or False
```

**Review by yogeshrnaik — DISMISSED** [review]

**Review by yogeshrnaik — APPROVED** [review]

**Comment by behrensger** [comment]
Hi Naik, 
you approved the PR. As far as I understand do you also have to do a merge. I cannot do the merge since I have no rights on your repository.
Kind Regards

**Comment by yogeshrnaik** [comment]
> Hi Naik,
> you approved the PR. As far as I understand do you also have to do a merge. I cannot do the merge since I have no rights on your repository.
> Kind Regards

@behrensger I have merged your PR. Thank you for your contribution.

**Pull request merged**